src/main/resources/log4j2.xml

```diff
@@ -15,38 +15,62 @@
 ~ See the license for the specific language governing permissions and
 ~ limitations under the license.
 -->
-<Configuration status="ERROR">
+<Configuration status="DEBUG">
     <Properties>
         <Property name="DEFAULT_HOME">$${env:JENKINS_HOME:-$${sys:JENKINS_HOME:-.}}</Property>
         <Property name="FALLBACK_LOG_DESTINATION">${DEFAULT_HOME}/logs/audit.log</Property>
         <Property name="AUDIT_LOG_DESTINATION">$${sys:auditFileName:-${FALLBACK_LOG_DESTINATION}}</Property>
         <Property name="IANA_ENTERPRISE_NUMBER">$${sys:enterpriseNumber:-.}</Property>
-        <Property name="APPENDER_LAYOUT_TYPE">$${sys:appenderType:-.}</Property>
+        <Property name="APPENDER_LAYOUT_TYPE">$${sys:appenderType:-jsonLayout}</Property>
+        <Property name="PORT">1854</Property>
+        <Property name="HOST">localhost</Property>
     </Properties>
+
     <Appenders>
         <Routing name="Routing">
-            <Routes pattern="${APPENDER_LAYOUT_TYPE}">
-                <Route>
-                    <RollingRandomAccessFile name="audit" fileName="${AUDIT_LOG_DESTINATION}" filePattern="${AUDIT_LOG_DESTINATION}.%d{yyyyMMdd_HHmmss}-%i.log">
+            <Script name="RoutingInit" language="Groovy"><![CDATA[
+            (System.properties['appenderType'] == "syslogLayout") ? "syslogLayout" : "jsonLayout";]]>
+            </Script>
+            <Routes>
+                <Route key="syslogLayout">
+                    <!--This has to be configurable-->
+                    <Syslog name="Audit Sys Logger"
```

Comment:
An interesting potential limitation I thought of here might be that this appender fails to start up because it can't connect to the configured syslog server, that might make the entire `<Routes/>` appender fail, though I'm not entirely sure.

Reply:
Turns out that shouldn't be a problem because the appender isn't created until a message matching the route is routed first.